src/main/kotlin/uk/gov/justice/digital/hmpps/makerecalldecisionapi/controller/RecommendationController.kt

```diff
@@ -113,11 +113,14 @@ internal class RecommendationController(
   suspend fun generateDntrDocument(
     @PathVariable("recommendationId") recommendationId: Long,
     @RequestBody documentRequestQuery: DocumentRequestQuery,
+    @RequestHeader("X-Feature-Flags") featureFlags: String?
   ): ResponseEntity<DocumentResponse> {
     log.info(normalizeSpace("Generate DNTR document endpoint for recommendation id: $recommendationId"))
 
+    val mapper = jacksonObjectMapper()
```

Comment:
Maybe we could move this block to a reusable class?

Reply:
Good idea! Done